boards/riscv/adp_xc7k_ae350/adp_xc7k_ae350_defconfig

```diff
@@ -13,6 +13,9 @@ CONFIG_SERIAL=y
 # GPIO driver options
 CONFIG_GPIO=y
 
+# SPI driver options
+CONFIG_SPI=y
+
```

Comment:
Don't y-select `CONFIG_SPI` by default. See this PR: https://github.com/zephyrproject-rtos/zephyr/pull/48574

Reply:
OK! I got it!

Thank a lot for your remind